pkgs/applications/video/davinci-resolve/default.nix

```diff
@@ -252,7 +256,28 @@ buildFHSEnv {
     ''
   }";
 
-  passthru = { inherit davinci; };
+  passthru = {
+    inherit davinci;
+    updateScript = lib.getExe (writeShellApplication {
+      name = "update-davinci-resolve";
+      runtimeInputs = [ curl jq yq common-updater-scripts ];
+      text = ''
+        set -o errexit
+        drv=pkgs/applications/video/davinci-resolve/default.nix
+        currentVersion="$(grep -E 'version = "[0-9]+\.[0-9]+\.[0-9]+"' $drv | cut -d '"' -f2)"
```

Comment:
I like the approach you are recommending since `nix eval` would be quite a bit longer:
```bash
nix eval --raw -I nixpkgs=./ nixpkgs#davinci-resolve-studio.davinci.version
```